app/features/receive/cashu-receive-quote-repository.ts

```diff
@@ -65,7 +69,30 @@ type CreateQuote = {
    *         Used for cross-account cashu token receives where the receiver chooses to claim a token to an account different from the mint/unit the token originated from, thus requiring a lightning payment.
    */
   receiveType: CashuReceiveQuote['type'];
-};
+  /**
+   * The amount of the token to receive.
+   */
+  tokenAmount?: Money;
```

Comment:
why do we need to have properties as optional here if we have union type below?

Reply:
I guess we can keep if you think it's helpful

Reply:
if we keep it I would update comment to say that it will be present only for receiveType token